src/components/RightPane/CoursePane/SearchForm/AdvancedSearch.tsx

```diff
@@ -250,18 +251,19 @@ class AdvancedSearch extends PureComponent<AdvancedSearchProps, AdvancedSearchSt
     };
 
     render() {
-        const { classes } = this.props;
-
         return (
             <>
-                <div onClick={this.handleExpand} className={classes.container}>
+                <Button
```

Comment:
Section code in the calendar popup is now a button; doesn't seem to significantly impact the UI. But I can style it more to make it seem "invisible" like before. (same for the section code in the right course pane)

Reply:
It's fine no need to fix the styling.